addons/main/script_macros.hpp

```diff
@@ -9,19 +9,18 @@
 #define DFUNC(var1) TRIPLES(ADDON,fnc,var1)
 #define DEFUNC(var1,var2) TRIPLES(DOUBLES(PREFIX,var1),fnc,var2)
 
-#define QFUNC(var1) QUOTE(DFUNC(var1))
-#define QEFUNC(var1,var2) QUOTE(DEFUNC(var1,var2))

```

Comment:
This only matters when compiling with debug enabled, so it's not to worrisome. But I'd say if you use QFUNC you would not want to compile in the debug wrappers. A good example is config attributes referring to a call back. You would end up with completely different behaviour (need a compile instead of just a getvar).